src/components/UserSelector.tsx

```diff
@@ -1,19 +1,45 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import { User } from '../types/User';
+
+interface Props {
+  users: User[];
+  activeUser: User | null;
+  setActiveUser: React.Dispatch<React.SetStateAction<User | null>>;
+  getUserPost: (userId: number) => void;
+}
+
+export const UserSelector: React.FC<Props> = ({
+  users,
+  activeUser,
+  setActiveUser = () => {},
+  getUserPost = () => {},
```

Comment:
I think you don't need to default these functions in the props argument. As I see, these props are required in the `Props` interface. So they should exist anyway.